src/components/TodoHeader/TodoHeader.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { KeyboardEventHandler } from 'react';
+import classnames from 'classnames';
+
+interface Props {
+  activeTodos: number
+  title: string
+  isProcessed: boolean
+  onAddTitle: (value: string) => void
+  onAddTodo: KeyboardEventHandler<HTMLInputElement>
+}
+
+export const TodoHeader: React.FC<Props> = ({
+  activeTodos, title, onAddTitle, isProcessed, onAddTodo,
+}) => {
+  return (
+    <header className="todoapp__header">
+      <button
+        type="button"
+        className={classnames(
+          'todoapp__toggle-all',
+          { active: !activeTodos },
+        )}
+        aria-label="button-toggle-all"
+      />
+
+      <form>
```

Comment:
better add the todo on submit (`onSubmit` in form)

Reply:
@etojeDenys Can you take a look at why adding a todo stopped working?